### PR TITLE
Create test

### PR DESCRIPTION
**Pull request by Minsamin**

### CODE REVIEW

**Review by Minsamin — COMMENTED** [review]
good job

**Comment by desarrollowebestudio** [comment]
Testing how Pull Requests work while following Bodgan's course about Git.

File: test [hunk 1]
```diff
@@ -0,0 +1 @@
+add a testFile to this project :p
```
Comment (desarrollowebestudio) [inline test:1]: Testing single line comments while following Bogdan's course about Git.

**Review by desarrollowebestudio — CHANGES_REQUESTED** [review]
This would be probably better inside a custom folder, instead of the root of the repository.

**Review by grb0 — APPROVED** [review]

File: test (hunk 1 above)
Comment (mikegithubber) [reply test:1]: Please create file in the folder

**Pull request closed without merging**